posix.mak

```diff
@@ -142,24 +142,24 @@ ALL_FILES = $(ALL_FILES_BUT_SITEMAP) $(DOC_OUTPUT_DIR)/sitemap.html
 
 # Pattern rulez
 
-$(DOC_OUTPUT_DIR)/%.html : %.dd $(DDOC)
+$(DOC_OUTPUT_DIR)/%.html : %.dd $(DDOC) $(DMD)
 	$(DMD) -c -o- -Df$@ $(DDOC) $<
 
-$(DOC_OUTPUT_DIR)/%.php : %.php.dd $(DDOC)
+$(DOC_OUTPUT_DIR)/%.php : %.php.dd $(DDOC) $(DMD)
 	$(DMD) -c -o- -Df$@ $(DDOC) $<
 
 $(DOC_OUTPUT_DIR)/% : %
 	@mkdir -p $(dir $@)
 	cp $< $@
 
-$(DOC_OUTPUT_DIR)/dmd-%.html : %.ddoc dcompiler.dd $(DDOC)
+$(DOC_OUTPUT_DIR)/dmd-%.html : %.ddoc dcompiler.dd $(DDOC) $(DMD)
 	$(DMD) -c -o- -Df$@ $(DDOC) dcompiler.dd $<
 
 ################################################################################
 # Rulez
 ################################################################################
 
-all : docs html
+all : docs html kindle pdf
```

Comment:
Well we split them at some point, because those depend on external tools.
https://github.com/D-Programming-Language/dlang.org/pull/518
I'm fine with calling `make -f posix docs html`, but this breaks the default target for almost anybody.